ocs_ci/ocs/resources/mcg.py

```diff
@@ -453,29 +455,45 @@ def _check_aws_credentials():
 
         return creds_request, aws_access_key_id, aws_access_key
 
-    def create_new_aws_connection(self, cld_mgr, conn_name=None):
+    def create_connection(self, cld_mgr, conn_name=None, platform=constants.AWS_PLATFORM):
```

Comment:
I would expect that no platform would be default, and one will need to chose, esp. when we are turning old AWS only function into proper multi platform one.

Reply:
Ok, removed default platform from this.